base_layer/wallet/src/output_manager_service/recovery/standard_outputs_recoverer.rs

```diff
@@ -90,17 +91,15 @@ where TBackend: OutputManagerBackend + 'static
                         &self.master_key_manager.rewind_data().rewind_blinding_key,
                     )
                     .ok()
-                    .map(|v| ( v, output ) )
+                    .map(|v| (v, output))
             })
-            //TODO: This needs some investigation. We assume Nop script here and recovery here might create an
```

Comment:
We do assume Nop scripts in recoverable outputs. If we want to recover other scripts we will need to cater for them when we write the code to produce them. I feel like that goes without saying and doesn't need to be a comment.

Reply:
I agree